tests/epyccel/test_arrays.py

```diff
@@ -226,9 +226,7 @@ def test_array_int32_1d_sub_augassign(language):
     assert np.array_equal( x1, x2 )
 
 @pytest.mark.parametrize( 'language', [
-        pytest.param("c", marks = [
-            pytest.mark.skip(reason="Numpy sum not yet implemented for C language"),
-            pytest.mark.c]),
+        pytest.param("c", marks = pytest.mark.c),
         pytest.param("fortran", marks = pytest.mark.fortran)
     ]
 )
```

Comment:
This mark should not be needed anymore. You can rely on the feature to provide the `language` argument

Reply:
Fixed! thank you!